### ISSUE TITLE
output "leelaz_version" is error.

### ISSUE THREAD

**Issue by HenryHo2006** [gcp/leela-zero#1105]
>Got new job: selfplay
Engine has started.
Infinite thinking time set.
net: 85c6f2ada691baf9ba942ecd1460a5229684ce16349be1fd792c26a64907cd0a.
Starting thread 16 on GPU 0
{
    "cmd": "selfplay",
    "hash": "85c6f2ada691baf9ba942ecd1460a5229684ce16349be1fd792c26a64907cd0a",
    "leelaz_version": "0.12",
    "options": {
        "noise": "true",
        "playouts": "0",
        "randomcnt": "30",
        "resignation_percent": "0",
        "visits": "3200"
    },
    "options_hash": "fd7105",
    "random_seed": "11662526477572650577",
    "required_client_version": "14"
}

"leelaz_version": "0.12"  is ERROR. the correct is 0.13

**Comment by isty2e** [comment]
#1096 

Maybe we should've used some other name for the field to prevent this kind of confusion...?

**Comment by Eddh** [comment]
https://github.com/gcp/leela-zero/issues/1096
https://github.com/gcp/leela-zero/issues/893
At this point we'll need to change the name of this variable.

**Comment by roy7** [comment]
https://github.com/gcp/leela-zero/issues/1096

@marcocalignano Since this is an endless battle we'll never win, can you add support to autogtp for either leelaz_version or required_leelaz_version to do the same thing, and the next time we have a required release after this version, we'll change the server to send "required_leelaz_version" instead?

**Comment by Eddh** [comment]
Maybe be more precise, "minimum_leelaz_version", otherwise people with 0.13 will ask why it says 0.12

**Comment by roy7** [comment]
@isty2e This came up the last time we had a new release too, we should probably have taken action back then. :)

**Comment by Eddh** [comment]
In the meantime maybe add some words about it in the webpage ?

**Comment by roy7** [comment]
I started the ball rolling here https://github.com/gcp/leela-zero/pull/1106

**Comment by Hersmunch** [comment]
I guess we're lucky no one asks about `"required_client_version": "14"` :)

**Comment by marcocalignano** [comment]
@roy7 just tell me what you need. Bur be aware this goes in /next and will be available with the next version (16) So there will be 5 more post like this before that and probably 2 after because people that do not want to read always find mistake that are not there.

**Comment by marcocalignano** [comment]
@roy7 name you variables!

**Comment by isty2e** [comment]
I suggest using "minimum_leelaz_version" and "minimum_autogtp_version" to minimize confusion, hopefully.

**Comment by gcp** [comment]
People will stop complaining if you enforce v15+0.13 in a day or two, but it would be good to rename the vars indeed.

**Comment by roy7** [comment]
@marcocalignano I did the first part here https://github.com/gcp/leela-zero/pull/1106/files. 

Lets go with "minimum_leelaz_version" and "minimum_autogtp_version" if the second really is the autogtp version. I used to think of it as like the "protocol" version. But of course any new protocol is all handled by new autogtp after all.

We just need to add support for both now, so whenever v16+0.14 is required down the road everyone has it and we change the variables on the server side too.
